mailbox/tools/quota-recompute/src/main/java/org/apache/james/mailbox/quota/task/RecomputeCurrentQuotasTaskAdditionalInformationDTO.java

```diff
@@ -110,20 +116,34 @@ public static AdditionalInformationDTOModule<RecomputeCurrentQuotasTask.Details,
     }
 
     private final String type;
+    private final long processedQuotaRoots;
+    private final ImmutableList<String> failedQuotaRoots;
     private final List<RecomputeSingleQuotaComponentResultDTO> recomputeSingleQuotaComponentResults;
     private final Optional<RunningOptionsDTO> runningOptions;
     private final Instant timestamp;
 
     public RecomputeCurrentQuotasTaskAdditionalInformationDTO(@JsonProperty("type") String type,
+                                                              @JsonProperty("processedQuotaRoots") long processedQuotaRoots,
+                                                              @JsonProperty("failedQuotaRoots") ImmutableList<String> failedQuotaRoots,
```

Comment:
It would be preferable to have `Optional<>` here to denote that fact that those could be missing.